src/js/gumhandler.js

```diff
@@ -80,22 +82,17 @@ GumHandler.prototype = {
     }
     this.gumPendingDialog_.close();
     this.gumErrorDialog_.close();
-    this.gumNoDeviceDialog_.close();
     this.gumStreamSuccessCallback_();
   },
 
   gotError_: function(error) {
     clearTimeout(this.firstUserCheck_);
     this.gumPendingDialog_.close();
-    if (error.name === 'DevicesNotFoundError') {
-      this.gumErrorDialog_.close();
-      this.gumNoDeviceDialog_.open();
-    } else if (!this.gumBypassed_) {
+    if (!this.gumBypassed_) {
       this.gumNoDeviceDialog_.close();
       this.gumErrorMessage_.innerHTML = error.name;
       this.gumErrorDialog_.open();
-    }
-    if (this.gumBypassed_) {
+    } else if (this.gumBypassed_) {
       var traceGumBypassed = report.traceEventAsync('getusermedia');
```

Comment:
Just noticed this now... you should use traceEventInstant instead of traceEventAsync.